monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/RepositoryZipController.java

```diff
@@ -177,12 +181,22 @@ public void run(InputStream fileInput) throws Exception {
                 }
             });
             zipOutput.closeEntry();
+            itemsCount++;
         }
 
         final List<RepositoryFolder> folders = folder.getFolders();
         for (RepositoryFolder subFolder : folders) {
             String name = subFolder.getName();
+            System.out.println("Directory: " + path + name + "/");
```

Comment:
Please use logger instead of System.out
(and please do a search in the complete PR for this)